index.html

```diff
@@ -156,19 +157,27 @@
     marker.title = "Your face";
     map.appendChild(marker);
   }
+  function letterToNum(letter) {
```

Comment:
Can we change this instead to make |period| always be a number? This seems like a rusty nail waiting for us to step on it.